ext4/tar2ext4/tar2ext4.go

```diff
@@ -272,10 +715,53 @@ func ConvertAndComputeRootDigest(r io.Reader) (string, error) {
 }
 
 // ConvertToVhd converts given io.WriteSeeker to VHD, by appending the VHD footer with a fixed size.
-func ConvertToVhd(w io.WriteSeeker) error {
+func ConvertToVhd(w io.WriteSeeker, alignToMB bool) error {
 	size, err := w.Seek(0, io.SeekEnd)
 	if err != nil {
 		return err
 	}
-	return binary.Write(w, binary.BigEndian, makeFixedVHDFooter(size))
+
+	diskSize := size
+	if alignToMB {
+		// diskPlusFooter is the disk size plus the size of the vhd footer to be added
+		diskPlusFooter := size + int64(vhdFooterSize)
```

Comment:
I'm not clear on why we're doing this. The VHD footer isn't really part of the "disk", in that it shouldn't be observable by the guest. So if the guest expects the disk to be MB-aligned, I don't think we should be including the footer size in that measurement.

Also, if we need to enforce this alignment, can we just seek the appropriate amount forward before calling `ConvertToVhd`? Adding a specific `alignToMB` parameter here is a breaking change to the API, and is kind of messy. Would prefer if we can try to keep different concerns (alignment and VHD footer) separated instead.

Reply:
We can't seek forward before calling "ConvertToVhd" as the code is written today. If we seek forward in "ConvertMultiple" without writing anything to disk at that location, when "ConvertToVhd" seeks the end of the disk, it won't be the correct location. 

I wasn't really sure what the best course of action was to account for that other than to make a new "ConvertToVhdWithSize" func that takes in the end size of the disk instead of seeking for it itself. If you're okay with that, I'll update it.

Reply:
Let's start with what we're trying to do here and why. Is there an alignment requirement coming from somewhere? Is it on the disk or the VHD file itself?

Reply:
Yes, there's a request that the disk be aligned. As you pointed out, I shouldn't be including the size of the footer, so I'm working on that fix.

Reply:
I'd like to understand the reasoning behind this requirement, and where it comes from.

Reply:
Discussed offline, MB alignment is needed on the disk (rather than the VHD file) and Kathryn will add it as part of the tool that produces images, rather than in package `tar2ext4`.